bench/bench.ml

```diff
@@ -640,6 +642,7 @@ let cmd =
   let doc = "Run all the benchmarks." in
   ( Term.(
       const run
+      $ Index_lib.Private.Logs.setup_term (module Mtime_clock)
```

Comment:
```suggestion
      const (fun () -> run)
      $ Index_lib.Private.Logs.setup_term (module Mtime_clock)
```

We usually try to keep the non-Cmdliner API independent of the idiosyncrasies required by the Cmdliner one.